### PR TITLE
add run export  to major version

### LINKED ISSUE #48: Please add run_exports !
https://abi-laboratory.pro/index.php?view=timeline&l=libuv

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #48.

Closes #48 

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,9 @@ source:
     - test-udp-join-1.31.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('libuv', max_pin='x.x') }}
```
Comment (isuruf) [inline recipe/meta.yaml:16]: Let's do major version?

Comment (hmaarrfk) [reply recipe/meta.yaml:16]: better than nothing! so yes

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Pull request merged**